web/static/js/application.js

```diff
@@ -346,13 +346,22 @@
             $('#project_input').on('change', function () {
                 var newProject = this.value;
 
+                // Show the spinner.
+                var $loader = $('span.loader');
+                $('label[for="project_input"]').append($loader);
+                $loader.removeClass('hidden');
```

Comment:
On one hand this makes sense because it's the project field that's triggering the AJAX, but to the user no behaviour changes with that field. I might instead show it on the affected fields like `#user_input` and `#article_input`. Maybe that's even weirder, though... especially for TopEdits, which has a username, namespace and project field! Three loading indicators at once?

Or we could _just_ show the loading indicator on the project field, and disable the affected inputs while the project is loaded. Does that make more sense?